composer.json

```diff
@@ -14,7 +14,7 @@
         "symplify/phpstan-extensions": "^9.2",
         "symplify/easy-coding-standard": "^9.2",
         "symplify/rule-doc-generator": "^9.2",
-        "rector/rector-src": "dev-main",
+        "rector/rector-src": "dev-allow-static-property-fetch as 0.11.28",
```

Comment:
@TomasVotruba this required as in rector patch https://github.com/rectorphp/rector-src/pull/322 fow allow `StaticPropertyFetch` on `ReflectionResolver::resolvePropertyReflectionFromPropertyFetch()`